packages/exchange-io-erc1888/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@energyweb/exchange-io-erc1888",
+  "version": "1.0.0",
+  "description": "ERC1888 withdwaral/deposit processing for exchange",
+  "author": "EnergyWeb DevHub GmbH; Aleksandr Marenin, aleksandr.amarenin@energyweb.com",
+  "homepage": "https://github.com/energywebfoundation/origin#readme",
+  "license": "ISC",
```

Comment:
We release packages under GPLv3. Check other packages for the correct license.

Reply:
done, thanks